config/initializers/flipper.rb

```diff
@@ -6,6 +6,14 @@
     Flipper.new(Flipper::Adapters::ActiveRecord.new)
   end
 end
+
+if Rails.env.production?
+  Flipper::UI.configure do |config|
+    config.banner_text = '⚠️ Production environment: be aware that the changes have an impact on the application. Please, read the how-to before: https://github.com/openfoodfoundation/openfoodnetwork/wiki/Feature-toggle-with-Flipper'
```

Comment:
Any chance we can add a translation key for this?

Reply:
It wouldn't harm but this is meant for the core team only though

Reply:
Why didn't I think of that? Maybe because FlipperUI itself isn't translated.
Anyway, I'v created an issue: #7463